src/app/routes/api/v3/admin/forms/admin-forms.form.routes.ts

```diff
@@ -197,3 +198,17 @@ AdminFormsFormRouter.put(
   '/:formId([a-fA-F0-9]{24})/start-page',
   AdminFormController.handleUpdateStartPage,
 )
+
+/**
+ * Retrieves the free sms counts used by a form's administrator
+ * @security session
+ *
+ * @returns 200 with the free sms counts
+ * @returns 401 when user does not exist in session
+ * @returns 404 when the formId is not found in the database
+ * @returns 500 when a database error occurs during retrieval
+ */
+AdminFormsFormRouter.get(
+  '/:formId([a-fA-F0-9]{24})/verified-sms/count/free',
+  SmsController.handleGetFreeSmsCountForFormAdmin,
```

Comment:
Now that it's nested in AdminForm router, should the controller function be in AdminFormController instead?

Reply:
sure! i agree with your point that it's easier to find also so will make the required changes